src/openrct2/network/NetworkBase.cpp

```diff
@@ -47,7 +47,7 @@
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-constexpr uint8_t kStreamVersion = 0;
+constexpr uint8_t kStreamVersion = 99;
```

Comment:
Do we want to keep this number, of change it before merging? (Making a note just in case.)

Reply:
I marked that commit as temp, just for testing, will revert when we are confident its good.